test/manifests/no-icon.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Twitter",
+  "short_name": "Twitter",
+  "start_url": "/?utm_source=web_app_manifest",
+  "display": "standalone",
+  "gcm_sender_id": "49625052041",
```

Comment:
nit: to harden test cases against future changes, I'd recommend limiting test data to the minimum needed to expose a specific behavior. In this case this'd mean removing: gcm_sender_id, gcm_user_visible_only, related_applications,...